ocaml/typing/typeopt.ml

```diff
@@ -197,20 +196,20 @@ let bigarray_type_kind_and_layout env typ =
   | _ ->
       (Pbigarray_unknown, Pbigarray_unknown_layout)
 
-let value_kind_of_value_layout layout =
-  match Layout.get_default_value layout with
+let value_kind_of_value_jkind jkind =
```

Comment:
Probably this function really takes a layout argument and shouldn't be renamed.

Reply:
Disagree here. Types are classified by jkinds, not layouts, and the arguments to this function are type classifiers.